build/.map

```diff
@@ -0,0 +1,79 @@
+# Scripts Map (Scripts Call Tree, shows the relations between the scripts).
+# Azure Pipelines documentation: https://docs.microsoft.com/en-us/azure/devops/pipelines/?view=azure-devops
+# See also: 
+#   QDK build pipeline wiki https://ms-quantum.visualstudio.com/Quantum%20Program/_wiki/wikis/Quantum-OKRs.wiki/328/QDK-build-pipeline
```

Comment:
This is not a public path and should not appear in this repo.

Reply:
Thanks, removing.